services/src/main/resources/db/changelog/gamification.db.changelog-1.0.0.xml

```diff
@@ -409,4 +409,14 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
             <column name="RULE_ID"/>
         </createIndex>
     </changeSet>
+    <changeSet id="1.0.0-35" author="exo-gamification">
```

Comment:
Is this change tested on MySQL, it seems not. Could you check please

Reply:
BTW, why this change exists in develop and not stable/2.3.x ? it due to a commit that is not backported apparently https://github.com/Meeds-io/gamification/commit/978945a52862c3eea93216ecd5ae88a589d73539

Reply:
> BTW, why this change exists in develop and not stable/2.3.x ? it due to a commit that is not backported apparently [978945a](https://github.com/Meeds-io/gamification/commit/978945a52862c3eea93216ecd5ae88a589d73539)

Yep, that's why I get a conflict when I tried to backport this fix. For MySQL SGDB, I tested it on maint-ft1 and I found it works